flink/v1.17/flink/src/main/java/org/apache/iceberg/flink/source/IcebergSource.java

```diff
@@ -426,7 +432,8 @@ public IcebergSource<T> build() {
         }
       }
 
-      contextBuilder.resolveConfig(table, readOptions, flinkConfig);
+      FlinkReadConf flinkReadConf = new FlinkReadConf(table, readOptions, flinkConfig);
+      contextBuilder.resolveConfig(flinkReadConf);
```

Comment:
Since we have changed to `ScanContext`, the modification here seems meaningless, what do you think?

Reply:
Correct!
Done